### DISCUSSION TITLE
Publish on Flathub

### DISCUSSION THREAD

**Discussion by benjamimgois** [rustdesk/rustdesk#5335] [Ideas]
Rustdesk already have a flatpak avaiable, and it works very well. But it really needs to be published on flathub so more people have access to it.

**Comment by xlionjuan** [comment]
And also Ubuntu Snap store or host a APT, RPM repository server.

**Comment by olumolu** [comment]
What is the issue why this app is not in flathub can @rustdesk elaborate the issue and explain if all apps can be in flathub why it is not in flathub what is the issue.
Either devlopers don't care about the users else there is technical issue.

**Reply by rustdesk** [reply]
DONT @ me please.

**Comment by rustdesk** [comment]
Close and lock to favor https://github.com/rustdesk/rustdesk/discussions/1537
